.github/actions/validate.yaml

```diff
@@ -0,0 +1,33 @@
+name: 'Validate'
+description: 'Validate kubernetes manifests using kustomize build'
+inputs:
+  kustomize-tag:
+    description: 'Kustomize version tag'
+    required: true
+    default: 'v4.5.1'
+  kustomize-sha:
+    description: 'Kustomize SHA'
+    required: true
+    default: 'b8e16b303d9aa210d135710efb63f39ec15ae2a5fb5dedf38698f512f95cd2bd'
+runs:
+  using: "composite"
+  steps:
+    - name: Kustomize build
+      run: |
+        set -e
```

Comment:
There is a kustomize action https://github.com/marketplace/actions/kustomize-github-action that colde potentially help out with this.
Do we want to jump in to the github action world or do we want it as similar to Azure DevOps as possible?

Reply:
That looks like a workflow. This is an an Action (smaller building block that can be used from a workflow). Also, antother difference is that this one finds all kusomization.yaml files and run a build for all of them. But it would be nice to re-use something

Reply:
Yes, you can use the action to download the binary if you want.
Then you can override the command that will be used and instead use your shell script.

But it's definitely **not** a must.